inventory/sample/group_vars/all.yml

```diff
@@ -10,6 +10,9 @@ system_timezone: "Your/Timezone"
 # interface which will be used for flannel
 flannel_iface: "eth0"
 
+# interface which will be used for kube-vip
+kube_vip_iface: "eth0"
```

Comment:
I don't think we should hardcode a default here.  `kube-vip` will determine one on its own.  I do think it would be nice to add it, but commented out.  Something like:

# uncomment kube_vip_iface to override the default kube-vip interface
# kube_vip_iface: "eth0"

Reply:
Hi @timothystewart6. Sure, adding a comment will also be nice